features/portal/DARTS_Case_Expiry.feature

```diff
@@ -0,0 +1,41 @@
+# TO DO: This feature only obfuscates events via Admin portal currently. 
+# To implement case expiry we would likely need to amend expiry date of case via DB client and then run automated job to expire case.
+@portal @portal_case_expiry @regression
+Feature: Case Expiry
+
+  @regression @sequential
+  Scenario: Case Expiry data creation
+    Given I create a case
+      | courthouse         | courtroom  | case_number | defendants     | judges          | prosecutors    | defenders    |
+      | HARROW CROWN COURT | A{{seq}}-1 | A{{seq}}001 | Def A{{seq}}-1 | JUDGE {{seq}}-1 | testprosecutor | testdefender |
```

Comment:
We could do with using a prefix for courtroom, case number etc that isn't already used, I think maybe this
```suggestion
      | HARROW CROWN COURT | J{{seq}}-1 | J{{seq}}001 | Def J{{seq}}-1 | JUDGE {{seq}}-1 | testprosecutor | testdefender |
```